### ISSUE TITLE
Bug Report- Site Visitors Don't Match Analytics

### ISSUE THREAD

**Issue by Braydenrud** [merchantprotocol/theadslab#17]
Issue Brief
The number of site visitors in Matomo, and google analytics don't match.
When was this bug experienced:
3/29/22

What is the purpose of this issue? Explain the background context. Please remember to search the issue tracker for your issue before posting a possible duplicate. It's better to enhance an existing issue with your experience than to create a second issue that will be immediately closed.
After installing the pixel on my site, I assumed it would also track visitors accurately. There is a large discrepancy between the values on Matomo and Google Analytics

Resources
--
Attach any screenshots, screencasts, customer information, chat transcripts, emails to support the debugging effort.
<img width="574" alt="Screen Shot 2022-03-29 at 5 11 01 PM" src="https://user-images.githubusercontent.com/102629776/160715731-8f4ea7b6-2a62-428a-b04f-36ed2b9def36.png">
<img width="1255" alt="Screen Shot 2022-03-29 at 5 11 17 PM" src="https://user-images.githubusercontent.com/102629776/160715734-2fc6dfa0-ae60-42d4-b3ae-5e839d0b5696.png">

Steps to reproduce
--
Listing of detailed steps needed to reproduce this error. Emphasis any steps that are critical to the reproduction of the bug.
1. Open Matomo and Google Analytics
2. Set date range and values to be the same
3. See if the number of site visitors is the same

Actual result
--
There are 5x the amount of site visitors on Google Analytics compared to Matomo

Expected result
--
The number of site visitors should match each other.

**Comment by jonathonbyrdziak** [comment]
I don't know why these numbers would be so wildly different at this time. I only have observations to add to your comments.

- In our system I don't see any traffic to your website other than what I believe to be you, yourself. All three and the only three visits on your account have the exact same IP address.
- I just visited the pages tracked in your account that the other user visited and I now show up as your fourth tracked visit.
- Our system DOES NOT track bots, google analytics DOES track bots.

Things to test and report back on for me:
- I would be curious to know upon further review if the analytics ID in the website matches the account you're looking at.
- What are all the links in google that people are visiting. I'd like to see a list of those pages, so that I could visit them and ensure that our pixel was installed correctly on those same pages.
- I would analyze the GA hits you have and see if there's some kind of trend that would lead us to a "cause"

Without a cause, we cannot fix anything.

**Comment by cliffsvr** [comment]
Hey Jonathon, did a quick check for my virtuallatinos account and I seem to have the same disparity in unique US visitor numbers. See screenshot belo
![Screenshot 2022-03-30 at 10 19 30](https://user-images.githubusercontent.com/102632679/160726911-ed08ac01-e25a-4a6e-b594-0c1d1c7189c1.png)
w:

**Comment by Braydenrud** [comment]
Jonathon, Yes the site visitors on these websites are rather low. I am trying to test it on my family's business sites just to see how to tool works and plan my fulfillment accordingly. 

We have made several sales off of our website since installing this pixel, it did not track any of those visitors either. 

I attached a couple of images you asked for. Could you clarify how I would check that the analytics id matches?
<img width="1032" alt="Screen Shot 2022-03-30 at 8 47 28 AM" src="https://user-images.githubusercontent.com/102629776/160850671-34d2b607-f6a2-42ac-adf1-b9d45bf87029.png">
<img width="864" alt="Screen Shot 2022-03-30 at 8 50 08 AM" src="https://user-images.githubusercontent.com/102629776/160850691-cdf15560-cbe0-4877-8980-4dac82b981ac.png">

**Comment by Braydenrud** [comment]
Also Here is the site traffic from the website builder. The site visitors again are much higher than on Matomo.
<img width="1102" alt="Screen Shot 2022-03-30 at 8 52 30 AM" src="https://user-images.githubusercontent.com/102629776/160851088-201662e0-9f8f-4998-a596-8ec9bc4ed207.png">

**Comment by jonathonbyrdziak** [comment]
> Hey Jonathon, did a quick check for my virtuallatinos account and I seem to have the same disparity in unique US visitor numbers. See screenshot belo
> ![Screenshot 2022-03-30 at 10 19 30](https://user-images.githubusercontent.com/102632679/160726911-ed08ac01-e25a-4a6e-b594-0c1d1c7189c1.png)
> w:

Yes this account has a competing matomo pixel installed and it's not going to be accurate. you need to uninstall the other matomo pixel

**Comment by jonathonbyrdziak** [comment]
Can you share your analytics with me "jonathonbyrd@gmail.com" and also let me know the SITEID of your matomo site

**Comment by cliffsvr** [comment]
Hi Jonathan, I have another website with the same issue as Braydenrud. Matomo is recognizing far few visitors than what the client's analytics (GA) is finding.
I've shown everything in this video.
https://www.loom.com/share/ae1367d661384b17a5405ba79e718901

I've just seen your above message. I'll ask the client to add you to their GA if that helps.
